### PR TITLE
Import Lux humanized data

### PR DESCRIPTION
**Pull request by cyrillustan**

### CODE REVIEW

File: src/dataHelpers.jl [hunk 1]
```diff
@@ -91,6 +91,32 @@ function importKav(; murine = true, c1q = false, IgG2bFucose = false, retdf = fa
 end
 
 
+""" Humanized mice data from Lux 2014 """
+function importHumanized(dataType)
+    df = CSV.read(joinpath(dataDir, "lux_humanized_CD20.csv"), delim = ",", comment = "#", type = String)
+
+    RIIA = Dict("R/R" => 'R', "H/H" => 'H', "H/R" => 'Z')
+    RIIIA = Dict("F/F" => 'F', "V/V" => 'V', "F/V" => 'Z')
```
Comment (aarmey) [inline src/dataHelpers.jl]: Can you get rid of any of these conversions just by reformatting the file?

**Comment by cyrillustan** [comment]
Can we assume "n.d" in data means 0?

**Comment by aarmey** [comment]
Stands for not determined, so we don't know the value.

**Comment by cyrillustan** [comment]
`lux_humanized_CD20.csv` is the original data and `lux_humanized_CD19.csv` is the updated one btw. I'll remove `lux_humanized_CD20.csv` if `lux_humanized_CD20.csv` is good

**Comment by cyrillustan** [comment]
Okay I'll need to change then.

**Comment by cyrillustan** [comment]
It should work now

**Pull request merged**